docs/interfaces/postgres.rst

```diff
@@ -545,7 +547,9 @@ CrateDB and we love to hear feedback.
 .. _pg_enum: https://www.postgresql.org/docs/10/catalog-pg-enum.html
 .. _pg_range: https://www.postgresql.org/docs/10/catalog-pg-range.html
 .. _pg_roles: https://www.postgresql.org/docs/10/view-pg-roles.html
+.. _pg_tables: https://www.postgresql.org/docs/14/view-pg-tables.html
 .. _pg_tablespace: https://www.postgresql.org/docs/13/catalog-pg-tablespace.html
+.. _pg_views: https://www.postgresql.org/docs/14/view-pg-views.html
```

Comment:
Should we change the links and reference v14 for all of them?

Reply:
Good point, but I'd prefer to use a dedicated commit for that

Reply:
Or also fine to do this here if @proddata volunteer to do that :)

Reply:
I thought about it, but also thought it make sense to reference the version, which was used for implementing it.

Should I change them to `current` or `v14`
i.e. `https://www.postgresql.org/docs/current/view-pg-views.html` or `https://www.postgresql.org/docs/14/view-pg-views.html`

Reply:
I'd suggest using `14` as otherwise it may not be ensured that the links will be correct in future.

Reply:
Changed the links to `14` (and checked if they are working)